views/index.pug

```diff
@@ -11,30 +11,38 @@ html
     meta(http-equiv="Content-Type", content="text/html; charset=utf-8")
     meta(name="viewport", content="width=device-width, initial-scale=1")
     title #{__('TITLE', community)}
+    link(href="//static.uqcs.org/bulma/main.css" rel="stylesheet" type="text/css")
     link(href="css/style.css", rel="stylesheet", type="text/css")
-    link(href="//fonts.googleapis.com/css?family=Lato:300,400,700,900,700italic|Open+Sans:700italic,400,600,300,700,800", rel="stylesheet", type="text/css")
   body
-    #wrapper
-      .main
-        .header
-          img(src="images/logo.svg", onerror="this.src='images/logo.svg.png'", class="logo")
-          h2 #{__('HEADER', community)}
-        .content
-          .information
+    section(class="section")
+      .container
+        .columns.is-centered.is-mobile
+          .column#body
+            .my-1#header
+              img(src="images/logo.svg", onerror="this.src='images/logo.svg.png'", id="logo")
+              h4(class="title is-4 my-5") #{__('HEADER', community)}
```

Comment:
Title doesn't need to mention UQCS or email since our full logo is directly above it and email is below. (I think this text is set elsewhere.)